style.css

```diff
@@ -0,0 +1,547 @@
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #f6f8fa;
+}
+
+.canopy {
+  width: 55px;
+  height: 16px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  list-style: none;
+}
+
+.back-dev {
+  width: 88px;
+  height: 16px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.num {
+  width: 29px;
+  height: 16px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.html {
+  width: 29px;
+  height: 16px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.button1 {
+  padding-top: 3px;
+  background-color: #fff;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  position: absolute;
+  width: 182px;
+  height: 36px;
+  left: -6px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-left: 44px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding-left: 2px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: -12px;
+}
+
+.button1:hover {
+  background-color: #9fbcf1;
+  color: rgb(243 243 245);
+}
+
+.buttonsee {
+  margin-bottom: 20px;
+}
+
+.buttonsee2 {
+  margin-top: 28px;
+}
+
+.texform {
+  height: 50px;
+  border-radius: 8px;
+}
+
+.texform2 {
+  height: 174px;
+  border-radius: 8px;
+}
+
+.works2 {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  gap: 37px;
+  padding: 10px;
+}
+
+.works2 > div {
+  background-color: rgb(80 147 223 / 80%);
+  text-align: center;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 36px;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.buttondir {
+  margin-right: 127px;
+}
+
+.button:hover {
+  background: rgb(78 145 231);
+}
+
+.aboutsection {
+  margin-left: 24px;
+}
+
+.wood {
+  width: 100px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  list-style: none;
+  margin-right: 240px;
+}
+
+.connectmywork {
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+  font-family: Poppines, sans-serif;
+  font-style: normal;
+  line-height: 24px;
+}
+
+.social-icons-mywork {
+  margin-right: 50px;
+  margin-top: 12px;
+  list-style: none;
+  display: flex;
+  gap: 20px;
+}
+
+.allsklil {
+  width: 295px;
+  height: 0;
+  font-family: 'Poppins:ital', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 0px;
+  color: #172b4d;
+}
+
+.make {
+  width: 113px;
+  height: 66.72px;
+  list-style: none;
+  margin-right: 240px;
+}
```

Comment:
As mentioned the previous reviewer, kindly I suggest you stick to design as much as possible to make your project look more professional. please fix the `width` to make the design like it is in the template. 😄